static/templates/admin/editFilenameForm.html

```diff
@@ -1,9 +1,8 @@
 <iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
-<form id="edit_filename_form" method="post" action="editlabel/<%= pid %>" enctype="multipart/form-data" target="upload_file_container" class="edit_form edit_filename_form">
-	
+<form id="edit_filename_form" class="edit_form edit_filename_form">
 	<div class="form_field">
 		<label>Filename</label>
-		<input name="label" size="40" value="<%= metadata.label %>"></input>
```

Comment:
Looking through the solr indexing code, it appears we aren't populating `label` anymore, so removing that was a good thing, and perhaps we should make a followup to remove the field.

Unfortunately the title and the filename for a file object are not necessarily going to be the same... The actual filename is stored in the solr datastream field, for example:
`original_file|image/jpeg|IMG_20200324_174901.jpg|jpg|3890854|urn:md5:788a541160b26039299647c9005b0767|`
it is the 3rd field in there. Perhaps pull the value from there? I believe this field will get updated after edit filename completes, but you might want to verify that.